### PR TITLE
feat(blacklist): save signature to user generated blacklist items

### PR DESCRIPTION
**Pull request by kenodressel**

### CODE REVIEW

File: migrations/_current.json [hunk 1]
```diff
@@ -872,17 +882,17 @@
...
                     "allowNull": false,
                     "field": "topics",
-                    "seqType": "Sequelize.ARRAY(Sequelize.TEXT)"
+                    "seqType": "Sequelize.ARRAY(Sequelize.STRING)"
                 },
                 "media": {
                     "allowNull": true,
                     "field": "media",
-                    "seqType": "Sequelize.ARRAY(Sequelize.TEXT)"
+                    "seqType": "Sequelize.ARRAY(Sequelize.STRING)"
```
Comment (thepiwo) [inline migrations/_current.json:895]: Why string

File: migrations/_current.json [hunk 2]
```diff
@@ -872,17 +882,17 @@
...
                     "allowNull": false,
                     "field": "title",
-                    "seqType": "Sequelize.TEXT"
+                    "seqType": "Sequelize.STRING"
                 },
                 "topics": {
                     "allowNull": false,
                     "field": "topics",
-                    "seqType": "Sequelize.ARRAY(Sequelize.TEXT)"
+                    "seqType": "Sequelize.ARRAY(Sequelize.STRING)"
```
Comment (thepiwo) [inline migrations/_current.json:890]: Why string

File: migrations/_current.json [hunk 3]
```diff
@@ -872,17 +882,17 @@
                 "title": {
                     "allowNull": false,
                     "field": "title",
-                    "seqType": "Sequelize.TEXT"
+                    "seqType": "Sequelize.STRING"
```
Comment (thepiwo) [inline migrations/_current.json:885]: Why string

File: migrations/_current.json [hunk 4]
```diff
@@ -93,6 +93,16 @@
...
                 },
+                "signature": {
+                    "allowNull": true,
+                    "field": "signature",
+                    "seqType": "Sequelize.STRING"
+                },
+                "challenge": {
+                    "allowNull": true,
+                    "field": "challenge",
+                    "seqType": "Sequelize.STRING"
```
Comment (thepiwo) [inline migrations/_current.json:104]: Is string always sufficient?

Comment (kenodressel) [reply migrations/_current.json:104]: Yes because the data in the challenge is hashed and only the hash is signed.

**Review by thepiwo — APPROVED** [review]

**Pull request merged**